src/PrestaShopBundle/Twig/LayoutExtension.php

```diff
@@ -102,6 +102,7 @@ public function getGlobals()
             'default_currency_symbol' => $defaultCurrency instanceof Currency ? $defaultCurrency->getSymbol() : null,
             'root_url' => $rootUrl,
             'js_translatable' => [],
+            '_rtl' => $this->context->getContext()->language->is_rtl ? '_rtl' : '',
```

Comment:
```suggestion
            'rtl_sufix' => $this->context->getContext()->language->is_rtl ? '_rtl' : '',
```

Please change the variable name, it's more understandable.